tests/test_jira_issue_adf_template.py

```diff
@@ -19,7 +19,7 @@ def test_create_issue():
         other_info="Some other info"
     )
     # Validate against Atlassian Document Format schema
-    jsonschema = requests.get("http://go.atlassian.com/adf-json-schema").json()
+    jsonschema = requests.get("https://unpkg.com/@atlaskit/adf-schema@latest/dist/json-schema/v1/full.json").json()
```

Comment:
The link http://go.atlassian.com/adf-json-schema should always redirect to the most up-to-date schema. I think it is better to keep it like that. Otherwise we might validate against an outdated schema.

Reply:
Wait - the link https://unpkg.com/@atlaskit/adf-schema@latest/dist/json-schema/v1/full.json also redirects to an updated schema. So it shouldn't matter. Or is there another reason for this change?

Reply:
The redirect does not work in the test. Have you tried running the test? I get 403 Forbidden and no redirect happens. However the redirect from `latest` works in the test.